packages/bpk-component-image/src/BpkImage.js

```diff
@@ -56,22 +75,23 @@ Image.defaultProps = {
   hidden: false,
 };
 
-class BpkImage extends React.Component {
-  constructor(props) {
-    super(props);
+class BpkImage extends React.Component<BpkImageProps> {
```

Comment:
I know this was here already, but can you import the name `Component` instead of accessing from `React` object